packages/kbn-user-profile-components/src/imported_types/authenticated_user.ts

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
These types should be removed once the security plugin is moved to its own package.